mzLib/Test/TestRawFileReader.cs

```diff
@@ -19,9 +17,6 @@ public sealed class TestRawFileReader
         [TestCase("testFileWMS2.raw", "a.mzML", "aa.mzML")]
         [TestCase("small.raw", "a.mzML", "aa.mzML")]
         [TestCase("05-13-16_cali_MS_60K-res_MS.raw", "a.mzML", "aa.mzML")]
-        /// <summary>
-        /// Tests LoadAllStaticData for ThermoRawFileReader
```

Comment:
Codemaid deletes method descriptions below the test cases. Could you add these back above `[Test]`?

Reply:
i didnt know that. i'm putting them back

Reply:
Yeah, it's not the greatest behavior on Codemaid's part in my opinion!